whisper/whisperv5/filter.go

```diff
@@ -176,7 +183,7 @@ func (f *Filter) MatchTopic(topic TopicType) bool {
 	return false
 }
 
-func isPubKeyEqual(a, b *ecdsa.PublicKey) bool {
+func IsPubKeyEqual(a, b *ecdsa.PublicKey) bool {
```

Comment:
I am working on the command-line tool, and I definitely need to export this function.